lib/task-items.js

```diff
@@ -24,7 +27,12 @@ module.exports = function(db) {
       defaultValue: Sequelize.NOW,
       allowNull: false
     },
-    lockedBy: { type: Sequelize.STRING }
+    lockedBy: { type: Sequelize.STRING },
+    siblings: {
+      type: Sequelize.ARRAY(Sequelize.STRING),
```

Comment:
Are we going to be adding siblings based on their primary key `auto_id`? If so, I suspect this line should read `type:  Sequelize.ARRAY(Sequelize.INTEGER)`.